packages/botonic-react/src/utils.js

```diff
@@ -77,3 +77,10 @@ export const scrollToBottom = (timeout = 100) => {
     setTimeout(() => (frame.scrollTop = frame.scrollHeight), timeout)
   }
 }
+
+export const getParsedAction = botonicAction => {
+  if (botonicAction.split('create_case:').length > 1) {
```

Comment:
in this case imho creating a temporary variable for "botonicAction.split('create_case:'" has value due 1) dry
2) for debugging, it wil be easy to view the value of the splitted string

Reply:
early return practice says to returns the "bad path" asap, and end up with the "happy path" (ie. reverse if condition)